ntp-admin/src/bin/ntp-admin.rs

```diff
@@ -0,0 +1,55 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! Executable program to run the Omicron NTP admin interface
+
+use anyhow::anyhow;
+use camino::Utf8PathBuf;
+use clap::Parser;
+use omicron_common::cmd::CmdError;
+use omicron_common::cmd::fatal;
+use omicron_ntp_admin::Config;
+use std::net::SocketAddr;
+use std::net::SocketAddrV6;
+
+#[derive(Debug, Parser)]
+#[clap(name = "ntp-admin", about = "Omicron NTP admin server")]
+enum Args {
+    /// Start the NTP admin server
+    Run {
+        /// Address on which this server should run
+        #[clap(long, action)]
+        address: SocketAddrV6,
+
+        /// Path to the server config file
+        #[clap(long, action)]
+        config_file_path: Utf8PathBuf,
+    },
+}
+
+fn main() {
+    if let Err(err) = oxide_tokio_rt::run(main_impl()) {
```

Comment:
Do we think this service really needs 128 worker threads, or should we maybe consider building a single-threaded runtime here?